pages/linux/arch-chroot.md

```diff
@@ -0,0 +1,24 @@
+# arch-chroot
+
+> Enhanced `chroot` command for Arch Linux installation.
+> More information: <https://man.archlinux.org/man/arch-chroot.8>.
+
+- Start an interactive shell in a new root directory:
+
+`arch-chroot {{path/to/new/root}}`
+
+- Specify the user to run as:
+
+`arch-chroot -u {{user}} {{path/to/new/root}}`
+
+- Run a command in a new root directory:
```

Comment:
We should clarify that this invocation executes the single command, and doesn't enter the chroot:
```suggestion
- Run a single command in a new root directory, and exit:
```

Reply:
Or perhaps use this opportunity to clarify the mental model of how it works:
```suggestion
- Run a custom command (instead of the default `bash`) in the new root directory:
```

WDYT @navarroaxel?

Reply:
I'm honestly not sure which of the two suggestions above is more helpful 😅 do you guys any preference?